### ISSUE TITLE
What about metadata file

### ISSUE THREAD

**Issue by olup** [marian-m12l/studio#295]
I am still implementing a CLI tool for lunii in golang, I'll post the repo around here once it has been published.

I think it would be extremely useful to have some metadata in the packs loaded on the lunii - I mean lunii's catalog is enough for official packs but side looaded pack then rely on a separate database that lives on the host, and it's not ideal.

I know you have been thinking the same from this comment 
https://github.com/marian-m12l/studio/blob/2aeb465f387ac200a83599c545d0d750c71c5014/core/src/main/java/studio/core/v1/writer/fs/FsStoryPackWriter.java#L44

So I will implement something like this for my cli, but I thought we could sync on what to use.

Top of my head, a simple `md` file, with a friendly format, either YAML or JSON (but I think json makes more sense) with at least the pack's complete uuid (the one that appears in the root's `.pi` file) and a title.

What do you think ?

**Comment by marian-m12l** [comment]
Yes that would be very nice. I added a few data in "raw" file format but was limited to stay compatible. No such limits with the "fs" file format. Did you end up implementing this?
